### PR TITLE
image encode decode

### LINKED ISSUE #74: Image Encryptor and Decryptor
## Description
Script to encrypt a given input image and also decrypt the encrypted image.
The script can generate a random noise image as a key and then use xor bitwise operation to encrypt the image.
And use a similar reverse process to decrypt it.

## Technologies and Languages

- Python

Using any other language and technology is also welcome.

## Aditional Work 
Make a GUI for this application

### PR DESCRIPTION
**Pull request by namanshah01**
# Description

Script that encodes and decodes a given image
<br>Image is converted to byte stream that is encoded by XOR-ing it with a randomly generated key. Which is stored into a text file
<br>Only with that key, the  reverse process on the text file will occur to convert it into an image again

Fixed #74 

## Type of change

_Please delete options that are not relevant._

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by bislara — CHANGES_REQUESTED** [review]
Perfect bro, nicely done. 🎉

- [ ] Put the project name, description and author in [readme](https://github.com/bislara/code-n-stitch/blob/master/Python/README.md) inside python folder. 

As soon as this is done. I will merge 🙌

File: Python/README.md [hunk 1]
```diff
@@ -55,4 +55,5 @@
 | 54   | [Website-cloner](/Python/website-cloner)  | Downloads all the files used in the website. | Neel Patel |
 | 55   | [Pinging Websites](/Python/pinging-websites)  | Ping a website or your own personal server and get instant emails whenever the server is down. | arnav-deep |
 | 56   | [PDF Encryptor](/Python/pdf_encryptor)  | Takes a non-password protected pdf file and encrypt it, requiring a password to unlock | LordChadiwala |
+| 56   | [Image Encode Decode](/Python/image-encoode-decode)  | Coverts image into a bytestream and encodes it, can only be decoded again with its key | namanshah01 |
```
Comment (bislara) [inline Python/README.md:58]: Change the number to 57 😅

Comment (namanshah01) [reply Python/README.md:58]: Oh sorry, that was silly :sweat_smile: 
I'll change right away

**Review by bislara — APPROVED** [review]

**Comment by bislara** [comment]
@all-contributors please add @namanshah01 for code, doc

**Pull request merged**